api/app/src/Migrations/Version292.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version292 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE court_order ALTER court_order_uid TYPE BIGINT USING court_order_uid::bigint');
+        $this->addSql('ALTER TABLE deputy DROP CONSTRAINT FK_28FA6B9FA76ED395');
+        $this->addSql('ALTER TABLE deputy ADD CONSTRAINT FK_28FA6B9FA76ED395 FOREIGN KEY (user_id) REFERENCES dd_user (id) ON DELETE CASCADE NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('ALTER TABLE document ALTER sync_attempts SET DEFAULT 0');
+        $this->addSql('ALTER TABLE pre_registration ALTER hybrid TYPE VARCHAR(6)');
+        $this->addSql('ALTER TABLE pre_registration ALTER client_firstname TYPE VARCHAR(100)');
```

Comment:
This potentially truncates fields. Have you checked whether there are any client_firstname values longer than 100 chars?

Reply:
Will double check, but there really shouldn't be anything that long as a first name.

Reply:
Current top is 21 characters